src/main/java/billgates/entities/User.java

```diff
@@ -1,29 +1,42 @@
 package billgates.entities;
 
+import java.util.List;
 import java.util.Objects;
 
 // singleton
 public class User {
 
     private static User instance;
-
     private int id;
     private String name;
     private String password;
     private int billID;
     private int currentBillID;
+    private List<AbstractEntry> currentBill;
```

Comment:
Though I added this instance variable, I haven't used it yet. 

I will think more about that and make some changes later on.

Reply:
As far as my thoughts, currentBillID and currentBill are both for Splitter exclusively, and it seems more abundant to keep track of the splitting bill than needed.

Moreover, is it possible for the user to create a currentBill, and let the bill point to whomever has access to it and needs to split it instead of having an instance (or more) in the user class?

If this question is not that urgent, we can save the discussion until the Splitter class implementation.

Reply:
> I don't think currentBillID and currentBill are for Splitter exclusively. I think they are used to reference the bill that should be displayed in the current GUI View.
For example, by default, if we are not using a Split Bill (At the moment), currentBill should always be the same value with billID.

In other words, currentBillID and currentBill are for presenters, and it depends on the user whether the bill should be split by other users, i.e., whether the bill goes to the Splitter.

> I think we can reserve this implementation for later, because implementing a splitter is a little bit more complicated than what we had first thought.

Understandable. Have a good night!